unit-9/exercise-5.js

```diff
@@ -0,0 +1,16 @@
+// a, b lần lượt là điểm thi giữa kỳ, điểm thi cuối kỳ
+const ranks = (a, b) => {
+  const medium = (a + b )/2
```

Comment:
```suggestion
  const medium = (a + b) / 2
```
medium là thông thường, bình thường. Chứ không phải là trung bình. Từ cần dùng là `average` trung bình cộng.